src/SwedbankPay.Sdk.Tests/Json/CustomAmountConverterTests.cs

```diff
@@ -19,13 +19,13 @@ public void CanDeSerialize_Amount()
         {
             //ARRANGE
 
-            var jsonObject = new JObject { { "xX123xxaddress", this.expectedAmount } };
+            var jsonObject = new JObject { { "xX123xxaddress", expectedAmount } };
```

Comment:
Please prefix fields with `this.`. :)

```suggestion
            var jsonObject = new JObject { { "xX123xxaddress", this.expectedAmount } };
```